driver-core/src/main/com/mongodb/operation/ChangeStreamOperation.java

```diff
@@ -46,16 +74,22 @@
  * @since 3.6
  */
 public class ChangeStreamOperation<T> implements AsyncReadOperation<AsyncBatchCursor<T>>, ReadOperation<BatchCursor<T>> {
+    private static final String RESULT = "result";
```

Comment:
Had to bring in code from AggregateOperation

Reply:
I see. Can it be DRY'd up a bit with an AggregationHelper?

Reply:
Done